### DISCUSSION TITLE
[Feature Request] Pause alerts from graph panel

### DISCUSSION THREAD

**Discussion by d-shi** [grafana/grafana#36997] [Alerting]
Right now it seems you can only pause alerts from the alerting page, which does not show you the dashboard name or the data source for the alert rules in the list. We have essentially duplicate dashboards/alerts for multiple kubernetes clusters, which means the alert rules in the alerting page have identical names, and there's no way to tell which one belongs to which cluster without clicking through to graph panel.

It would be really nice to be able to click the heart icon on the graph panel itself to configure the alert rule status.

**Comment by bergquist** [comment]
makes sense.

**Comment by raags** [comment]
Just to add to this / clarify:
- Namespace by dashboard name in the alerting page
- A visual cue to identify if an alert is paused (The panel has no indication if its alert is paused)
- Ability to pause (or silence with the silence feature) from the panel itself

**Comment by orojina** [comment]
@d-shi   In case you find this useful, and as a temporary solution,  I created a script to list  alerts with their statuses and the dashboard they belong to. Listing several alerts can be done with:

`grafana-cli alerts | grep -i <your_filter_cliteria>` 

![image](https://user-images.githubusercontent.com/11845190/107563867-32beaf00-6ba7-11eb-8d2f-92f53889c51c.png)

Adding the watch command into the mix will give you and your team an auto-refreshed view of your alerts

https://github.com/orojina/grafana-cli

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82983. Feel free to continue the discussion around this there. Thank you!
